WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -337,6 +337,9 @@ extension PostCoordinator: Uploader {
             }
 
             postsAndActions.forEach { post, action in
+                if action != .nothing, let status = post.status {
+                    self.trackAutoUpload(action: action, status: status)
+                }
```

Comment:
What do you think about moving these checks to the `trackAutoUpload` method?

It's a logic related to the track and this will make this call cleaner, without worrying about the `action` or if we have a `post.status`.

Reply:
Hey Leandro, I'll follow your suggestion cause i like that approach as well though I thought it would be better than having an optional parameter